src/components/sdk/MeetingControls/AudioInputControl.tsx

```diff
@@ -21,18 +21,21 @@ interface Props {
   mutedIconTitle?: string;
   /** Title attribute for the icon when unmuted, it defaults to `Microphone` */
   unmutedIconTitle?: string;
+  /** A boolean that determines whether or not to include additional sample audio devices to the control bar */
+  appendAdditionalDevices?: boolean;
 }
 
 const AudioInputControl: React.FC<Props> = ({
   muteLabel = 'Mute',
   unmuteLabel = 'Unmute',
   mutedIconTitle,
-  unmutedIconTitle
+  unmutedIconTitle,
+  appendAdditionalDevices = true,
 }) => {
   const meetingManager = useMeetingManager();
   const { muted, toggleMute } = useToggleLocalMute();
   const audioInputConfig: DeviceConfig = {
-    additionalDevices: true,
+    additionalDevices: appendAdditionalDevices,
```

Comment:
Doing so will remove the ability to select `null` device from `AudioInputControl`, `AudioInputVFControl` as well as `VideoInputControl` due to below reason:
1. These controls depend on `AudioInputProvider`, `VideoInputProvider` respectively.
2. Here we add the additional devices which also contain 'None'. [AudioInputProvider](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/providers/DevicesProvider/AudioInputProvider.tsx#L153-L160) and [VideoInputProvider](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/providers/DevicesProvider/VideoInputProvider.tsx#L111-L120). These both pick the additional devices from [here](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/constants/additional-audio-video-devices.ts).The 'None' is used as `null` device across library I think, so we cannot skip it.

We have to refactor this logic to at least include 'None' mapping.

Reply:
Yeah but this is for UI control right. I don't think most apps need to select None as an option (especially for UI).

Reply:
Another option is instead of boolean allow them to pass in the list of additional devices they want to.

Reply:
The problem with passing a list of devices is that the selected device ends up being interpreted by our code:

https://github.com/aws/amazon-chime-sdk-component-library-react/blob/d7d7a0fbe7e1789066bd448b41c8e6b0f9233180/src/utils/device-utils.ts#L31

and so the list of additional devices has to be part of a restricted set.

Reply:
Yeah, perhaps we can separate 'None' from the "additional devices"?